libbpf-rs/src/perf_buffer.rs

```diff
@@ -142,6 +161,49 @@ impl<'a, 'b> PerfBufferBuilder<'a, 'b> {
         }
     }
 
+    unsafe extern "C" fn call_event_cb(
```

Comment:
Can you please add `#[deny(unsafe_op_in_unsafe_fn)]` here and then add individual `unsafe` blocks?

Reply:
Done